api/spec/v1.yaml

```diff
@@ -921,6 +921,31 @@ paths:
                 $ref: '#/components/schemas/TxVolumeList'
         <<: *common_error_responses
 
+  /{layer}/stats/active_accounts:
+    get:
+      summary: |
+        Returns a (sliding) timeline of the recorded daily unique active accounts for
+        either consensus or one of the paratimes.
+      parameters:
```

Comment:
Will add. Although I think what you mean is actually `window_step_seconds`? Because the window size will currently always be 24 hours, even if the step is smaller - we only compute the unique accounts over the 24 hours intervals.

> Hum, makes me realize we don't probably limit the size of the output in any way right now. Oh well, we can fix that later.

Yeah, likely good idea to handle this in a unified way for all paginated endpoints.